src/cmd-buildextend-metal

```diff
@@ -185,6 +182,23 @@ if [ "${rootfs_type}" = "ext4verity" ]; then
     BLKSIZE="$(getconf PAGE_SIZE)"
 fi
 
+disk_args=()
+qemu_args=()
+# SecureExecution extra stuff
+if [[ $secure_execution -eq "1" ]]; then
+    disk_args+=("--with-secure-execution")
+    if [ -z "${genprotimgvm}" ]; then
+        qemu_args+=("-drive" "if=none,id=hostkey,format=raw,file=$hostkey,readonly=on" \
+                            "-device" "virtio-blk,serial=hostkey,drive=hostkey")
+    else
+        genprotimg_img="$(mktemp -p "${PWD}" secex-XXXXXX.img)"
```

Comment:
We're not cleaning up this file afterward and we're not using a consistent name, so the files will accumulate.

Should we be using the usual tmpdir instead of `$PWD`?

Reply:
Since `$PWD` should point to `tmp/build.qemu` at this point, i think the correct way would be to make the filename consistent. Because if i remember the implementation correctly that directory is nuked every time a new build is done.

Reply:
I changed the file name to be consistent, let me test it to make sure it works as intended.

Reply:
Works as expected, `tmp/build.qemu` is deleted after a successful run.